js/initcontext.go

```diff
@@ -1,254 +1,37 @@
 package js
```

Comment:
There's not much left in this file anymore, besides some helpers. Can we move those to where they make sense and remove this file? Same for `initcontext_test.go`.

We still refer to `InitContext` throughout the codebase, and we might want to rename some of those as well. Though I suppose we can still refer to the "init context" without having a direct implementation of it.

Reply:
I kind of prefer not to make this bigger.

I will likely do more PRs (soon :tm:) around this code. Some of it will *hopefully* rip all the modules related parts and put it in a single place to be used in this package as well as in the tc39 test and even for the other *actual* module tests ;) 

I also *specifically* left the `open` and `require` implementation in it as ... that is what is currently in the initcontext . I could probably move some of the other helpers around but IMO that will just make the PR harder to review :shrug:

Reply:
I don't think this change would make things much harder to review, but alright, it's fine as is.